string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,20 @@
+/* exported isAnagram */
+function isAnagram(firstString, secondString) {
+  var string1 = firstString.split('');
+  var string2 = secondString.split('');
+  string1.sort();
+  string2.sort();
+  var newString1 = '';
+  var newString2 = '';
+  for (var i = 0; i < string1.length; i++) {
+    if (string1[i] !== ' ') {
+      newString1 += string1[i];
+    }
+  }
```

Comment:
Once the strings are sorted all of the spaces will be at the beginning of the strings. Maybe there is a string method that will `trim` the spaces for the ends of a string?